src/pages/index.svelte

```diff
@@ -1,7 +1,164 @@
-<script lang='typescript'>
-const content: string = 'index'
+<script>
+import Datatable from '../components/Datatable/index.js'
+import { Card, Checkbox, CustomCard } from '@silintl/ui-components'
+
+// TODO: update this to be dependent on backend endpoint
+let selected = []
+let loading = false
+let shownMenu
+let clickedOnVert = false
+let exampleItems = [
+  {
+    id: 1234,
+    item_name: "Laptop",
+    recent_activity: "Awaiting approval",
+    accountable_person: "Bob Smith",
+    cost: 943,
+    premium: 18.86,
+    type: "Mobile",
+  },
+  {
+    id: 5678,
+    item_name: "Saxophone",
+    recent_activity: "Added a month ago",
+    accountable_person: "Mary Smith",
+    cost: 54,
+    premium: 1.08,
+    type: "Mobile",
+  },
+  {
+    id: 9101,
+    item_name: "Couch",
+    recent_activity: "Removed last week",
+    accountable_person: "Jim Smith",
+    cost: 943,
+    premium: 18.86,
+    type: "Stationary",
+  }
+]
+
+const handleChecked = id => {
+  selected.push(id)
+  console.log(selected)
+}
+const handleUnchecked = id => {
+  selected = selected.filter(val => val != id)
+  console.log(selected)
+}
+const handleMoreVertClick = id => {
+  if (shownMenu == id) shownMenu = null
+  else shownMenu = id
```

Comment:
Let's use curly braces around the contents of all `if` blocks (here and elsewhere):
```suggestion
  if (shownMenu == id) {
    shownMenu = null
  } else {
    shownMenu = id
  }
```